lib/pure/times.nim

```diff
@@ -2067,7 +1673,7 @@ template formatValue*(result: var string; value: Time, specifier: string) =
 
 proc parse*(input: string, f: TimeFormat, zone: Timezone = local(),
     loc: DateTimeLocale = DefaultLocale): DateTime
-    {.raises: [TimeParseError, Defect].} =
```

Comment:
see https://github.com/nim-lang/Nim/pull/16724#issuecomment-869899201

Reply:
Does that mean I should undo this change (also see https://github.com/nim-lang/Nim/pull/18378#issuecomment-869839791)?

Reply:
unless you can figure out a way that works during bootstrap, I don't see another way.

you can test locally whether it works by building `bin/nim_csources_a8a5241f9475099c823cfe1a5e0ca4022ac201ff c compiler/nim`

Reply:
So I also can't use `importutils.privateAccess`/`{.use.}`?

Reply:
you mean {.all.}, not {.use.} ? indeed. It can be used in code that's not used during compiler bootstrap or needs `when defined(nimHasHintAll)` or any recent such condsyms symbol